plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPathTest.java

```diff
@@ -31,38 +32,53 @@
 import com.google.cloud.tools.eclipse.appengine.libraries.model.Library;
 
 public class BuildPathTest {
+  
+  private final List<Library> libraries = new ArrayList<>();
```

Comment:
I see tests add elements, and I don't see they clear the list after use. Will there be a possibility of test failure if we re-order tests? Instantiating it in `@Before` or clearing it in `@After`?